projects/core-person-record-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integration/delius/entity/ReligionHistory.kt

```diff
@@ -0,0 +1,32 @@
+package uk.gov.justice.digital.hmpps.integration.delius.entity
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import org.hibernate.annotations.Immutable
+import java.time.LocalDate
+
+@Immutable
+@Entity
+@Table(name = "offender_religion_history")
+class ReligionHistory(
+    @Id
+    @Column(name = "offender_religion_history_id")
+    val id: Long = 0,
+
+    @Column(name = "offender_id")
+    val offenderId: Long = 0,
```

Comment:
Minor thing - but we usually rename this to something like `personId`, as the term `offender` was scrubbed from most of Delius (although it still appears in the DB tables/columns).
```suggestion
    val personId: Long = 0,
```

Reply:
fixed